### PR TITLE
News article stylesheet

### PR DESCRIPTION
**Pull request by ptomato**
https://phabricator.endlessm.com/T17401

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -308,12 +310,44 @@ class NewsArticle extends BaseAsset {
...
         }
 
+        let quote_svg_path = path.join(__dirname, './assets/quote.svg');
+        let quote_decoration_xml = fs.readFileSync(quote_svg_path, 'utf8');
+        let quote_decoration = cheerio.load(quote_decoration_xml, {
+            xmlMode: true,
+        });
+        cheerio('blockquote', this._body).prepend(`<div class="quote-decoration">
+            ${quote_decoration.html('svg')}
+        </div>`);
```
Comment (sgnn7) [inline lib/index.js]: I'm a little confused by this -are we assuming that everything is surrounded by quotes?

Comment (ptomato) [inline lib/index.js]: This should add the quote decoration as the first child of every `<blockquote>` element.

Comment (sgnn7) [inline lib/index.js]: I see what you mean - I thought it was prepending blockquote on the body with nothing on it. Thanks.

**Comment by sgnn7** [comment]
:+1:

**Pull request merged**